catalog/pipeline/fbc-release/0.1/README.md

```diff
@@ -0,0 +1,14 @@
+# FBC Release Pipeline
+
+FBC Release Pipeline
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| snapshot | The snapshot in JSON format | No | - |
```

Comment:
The description in line 10 capitalizes the CRD name, so I think it should be `Snapshot` in the description (parameter name still fine)